src/interfaces/Capabilities.ts

```diff
@@ -38,6 +38,14 @@ export enum MatrixCapabilities {
     * @deprecated It is not recommended to rely on this existing - it can be removed without notice.
     */
     MSC4039UploadFile = "org.matrix.msc4039.upload_file",
+    /**
+    * @deprecated It is not recommended to rely on this existing - it can be removed without notice.
+    */
+    MSC4157SendDelayedEvent = "org.matrix.msc4157.send.delayed_event",
+    /**
+    * @deprecated It is not recommended to rely on this existing - it can be removed without notice.
```

Comment:
Okay so we indeed use deprecated as a: "can be removed without notice." disclaimer.